model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -229,6 +230,14 @@ case class SubmissionReport(
   workflows: Seq[SubmissionValidationEntityInputs]
 )
 
+case class RetriedSubmissionReport(
+  originalSubmissionId: String,
+  submissionId: String,
+  submissionDate: DateTime,
+  submitter: String,
+  status: SubmissionStatus,
+)
```

Comment:
I suggest adding to this:
* the retry type (failed, aborted, failed+aborted)
* list of workflows, similar to `SubmissionReport.workflows`